hoover/search/es.py

```diff
@@ -43,13 +64,59 @@ def get(collection_id, doc_id):
         id=doc_id,
     )
 
-
-def search(query, fields, highlight, collections, from_, size, sort, aggs):
+def _get_indices(collections):
     from .models import Collection
     indices = ','.join(
         c.index for c in
         Collection.objects.filter(name__in=collections)
     )
+    return indices
+
+@convert_es_errors
+def batch_count(query_strings, collections, aggs=None):
+    def _serialize(object):
+        if object:
+            return json.dumps(
+                object,
+                separators=(',', ':'),
+            ).replace('\n', '')
```

Comment:
Why `replace` here? I'm pretty sure the JSON document is guaranteed to have no newlines inside. And you can safely leave the default separators in place even if they add a bit of extra whitespace.